gtsam/gtsam.i

```diff
@@ -2166,6 +2166,12 @@ virtual class NoiseModelFactor: gtsam::NonlinearFactor {
   Vector whitenedError(const gtsam::Values& x) const;
 };
 
+#include <gtsam/nonlinear/CustomFactor.h>
+virtual class CustomFactor: gtsam::NoiseModelFactor {
+  CustomFactor();
+  CustomFactor(const gtsam::SharedNoiseModel& noiseModel, const gtsam::KeyVector& keys, const gtsam::CustomErrorFunction& errorFunction);
```

Comment:
can you add a documentation/comment here about the args of `CustomErrorFunction`, just so that anyone using gtsam.i as an API reference doesn't have to go digging around to figure out how to use it?

Reply:
This should be add-able via the docstring mechanism. That way, the editor shows it. Adding usage docs should go in the cpp file and not in the .i file imo

Reply:
done